src/main/java/frc/robot/commands/AutoScoreAndBalance.java

```diff
@@ -18,12 +25,19 @@ public class AutoScoreAndBalance extends SequentialCommandGroup {
               Cal.SwerveSubsystem.MAX_LINEAR_SPEED_METERS_PER_SEC,
               Cal.SwerveSubsystem.MAX_LINEAR_ACCELERATION_METERS_PER_SEC_SQ));
 
-  public AutoScoreAndBalance(boolean isFirstPath, Lift lift, DriveSubsystem drive) {
-    // TODO update this since scoreGamePiece and doneScoring are stubbed
+  public AutoScoreAndBalance(boolean isFirstPath, Lift lift, DriveSubsystem drive, ScoringLocationUtil scoringLocationUtil) {
     addCommands(
-        new RunCommand(lift::scoreGamePiece, lift).until(lift::doneScoring),
-        drive.followTrajectoryCommand(traj, isFirstPath),
-        new AutoChargeStationBalance(drive));
+      new InstantCommand(()->scoringLocationUtil.toggleMiddleGrid()), //is this needed??
```

Comment:
I can't give you the answer to this because I don't know, at least not off the top of my head. But think about what is happening within this command. *Should* it be scoring in the middle grid? Also remember this function toggles it, so think about the starting state of the variable. (Since this is running in autonomous and is thus before any human input, the starting state should always be the same.)

Whatever the answer ends up being, we should know whether it is or isn't necessary before merging to main. The only unknowns would be if it requires testing the robot, which I don't think is the case here.

Reply:
Since this auto has the robot already at the scoring location, I think we can just totally ignore the scoring util for this auto. We only really need it for automatically driving to a location, which we don't need to do for this one

Reply:
Ok, I will remove it.